src/dotnet/CommonLocalizableStrings.resx

```diff
@@ -640,4 +640,10 @@ setx PATH "%PATH%;{0}"
   <data name="FormatVersionIsMissing" xml:space="preserve">
     <value>Format version is missing. This tool may not be supported in this SDK version. Please contact the author of the tool.</value>
   </data>
-</root>
+  <data name="MoreThanOnePackagedShimAvailable" xml:space="preserve">
+    <value>More than one packaged shim available, they are {0}</value>
```

Comment:
Perhaps just `More than one packaged shim is available: {0}.`

Reply:
and also @KathleenDollard for using facing string review.

Reply:
Changed as suggested